packages/ui/app/src/api-page/endpoints/EndpointContent.tsx

```diff
@@ -85,11 +87,24 @@ export const EndpointContent = React.memo<EndpointContent.Props>(function Endpoi
         [titleHeight]
     );
     const [selectedErrorIndex, setSelectedErrorIndex] = useState<number | null>(null);
+    const selectedError = selectedErrorIndex == null ? null : endpoint.errors[selectedErrorIndex] ?? null;
```

Comment:
nbd but everywhere else in the UI we never use null / always use undefined

Reply:
cool will switch to `undefined` for consistency